packages/@okta/vuepress-theme-prose/assets/css/components/_footer.scss

```diff
@@ -76,4 +76,251 @@ footer {
+        }
+
+        span.link-list--text {
+          display: none;
+        }
+      }
+    }
+
+    .contact {
+      @include media('<desktop') {
+        flex: 1 0 30%;
+        order: 1;
+        margin-bottom: 46px;
+      }
+
+      @include media('<tablet') {
+        flex: 1 0 50%;
+      }
+
+      @include media('<phone') {
+        flex: 1 0 100%;
+      }
+    }
+
+    .more {
+      @include media('<desktop') {
+        flex: 1 0 30%;
+        order: 2;
+        margin-bottom: 46px;
+      }
+
+      @include media('<tablet') {
+        flex: 1 0 50%;
+      }
+
+      @include media('<phone') {
+        flex: 1 0 100%;
+      }
+    }
+
+    .websites {
+      flex: 0 0 280px;
+
+      @include media('<desktop') {
+        flex: 1 0 40%;
+        order: 3;
+        margin-bottom: 46px;
+      }
+
+      @include media('<tablet') {
+        flex: 1 0 100%;
+      }
+
+      .website {
+        display: flex;
+        flex-direction: column;
+
+        & + .website {
+          margin-top: 40px;
+        }
+
+        .description {
+          font-size: 16px;
+          line-height: 1.7em;
+          letter-spacing: 0.01em;
+          color: $redesign_white;
+          margin-top: 6px;
+        }
+      }
+    }
+
+    .copyright {
+      flex: 0 0 180px;
+      display: flex;
+      justify-content: flex-end;
+      align-items: flex-end;
+      
+
+      @include media('<desktop') {
+        order: 5;
+        flex: 1 0 100%;
+        justify-content: center;
+      }
+
+      span {
+        font-style: normal;
+        font-weight: 600;
+        font-size: 16px;
+        line-height: 1.7em;
+        letter-spacing: 0.01em;
+        color: $redesign_white;
+        opacity: 0.9;
+      }      
+    }
+  }
+
+  a.link {
+    font-size: inherit;
+    line-height: 1.3em;
+    color: $redesign_white;
+
+    &:hover {
+      text-decoration: underline;
+    }
+
+    &--small {
+      font-size: 14px;
+    }
+
+    &--red {
+      color: $link_default;
+    }
+
+    &--bold {
+      font-weight: bold;
+    }
+
+    &--uppercase {
+      text-transform: uppercase;
+    }
+
+    &--underline {
+      text-decoration: underline;
+    }
+
+    &--spacing-large {
+      letter-spacing: 1.5px;
+    }
+
+    &[target='_blank']:not([href^='mailto:'])::after {
+      content: '';
+      display: inline-block;
+      width: 1em;
+      height: 1em;
+      margin-left: 0.2em;
+      background-image: url('data:image/svg+xml;utf8,<svg width="9" height="9" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M8.446 0H5.465c-.306 0-.643.337-.643.643s.337.643.643.643h1.607L4.178 4.038a.55.55 0 000 .784.55.55 0 00.785 0L7.715 1.93v1.607c0 .306.337.643.642.643.306 0 .643-.337.643-.643V.555A.555.555 0 008.446 0z" fill="%23dd2864"/><path d="M6.429 7.714H1.286V2.571h1.928L4.5 1.286H.643c-.263 0-.643.38-.643.643v6.428C0 8.68.321 9 .643 9h6.429c.321 0 .642-.321.642-.643V4.5L6.43 5.786v1.928z" fill="%23dd2864"/></svg>');
```

Comment:
Any chance we could use an SVG file instead of embedding the data inside the stylesheet?